### ISSUE TITLE
Sample fails due to file size > 64KB

### ISSUE THREAD

**Issue by acinep** [Azure/azure-relay-aspnetserver#7]
I believe there is an issue with the sample MVC app - files over 64KB silently fail (200 response, but no content).

As an example - a request for `bootstrap.js` fails, but a request for `bootstrap.min.js` works ok.

This [page](https://docs.microsoft.com/en-us/azure/service-bus-relay/relay-faq) mentions a 64KB limit on message size for NetOnewayRelayBinding and NetEventRelayBinding relays?

**Comment by dlstucki** [comment]
@clemensv, this is built on top of Microsoft.Azure.Relay.dll right? That it supposed to handle larger payloads by doing the rendezvous under the covers. I wonder why this isn't working.

**Comment by dlstucki** [comment]
Will you ensure that the latest preview version of [Microsoft.Azure.Relay](https://www.nuget.org/packages/Microsoft.Azure.Relay/2.0.0-preview1-20180523) is referenced and used?

**Comment by acinep** [comment]
@dlstucki Yep - I am seeing the same behavior with `2.0.0-preview1-20180523` and `1.1.0-preview`.  I will run through it with fiddler or wireshark and see if I can see where the rendezvous is falling down.
